lib/src/components/NxVulnerabilityCounter/NxVulnerabilityCounter.scss

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@use '../../scss-shared/nx-text-helpers';
+@use '../../scss-shared/nx-dark-mode-helpers';
+
+.nx-vulnerability-counter {
+  align-items: center;
+  box-sizing: content-box;
+  border: 2px solid var(--nx-color-vulnerability-unspecified);
+  border-left-width: 8px;
+  color: var(--nx-color-vulnerability-level-text);
+  display: inline-flex;
+  flex-direction: row-reverse;
+  font-size: var(--nx-font-size-s);
+  justify-content: flex-end;
+  height: 28px;
```

Comment:
Could a shared variable be used to ensure this height remains the same as:
https://github.com/sonatype/sonatype-react-shared-components/blob/da283bc73482926ab19e666cf56ea54115af8187/lib/src/components/NxThreatCounter/NxThreatCounter.scss#L21
?

Reply:
I created new variables in d3bce8bc1855082ce4163e21d0a2367b2457e319, let me know if it that implementation is ok

Reply:
> I created new variables in [d3bce8b](https://github.com/sonatype/sonatype-react-shared-components/commit/d3bce8bc1855082ce4163e21d0a2367b2457e319), let me know if it that approach is ok

Based on https://github.com/sonatype/sonatype-react-shared-components/pull/1200#discussion_r1794439392 that change would have to be reverted.
Sorry for the inconvenience 🙏

Reply:
Reverted changes on [987ad66](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/987ad662a8f2bca6b01578f882d1b64cdac6eb3b)